### ISSUE TITLE
Better pptx file conversion (animated effect and embedded video)

### ISSUE THREAD

**Issue by hiroshisuga** [bigbluebutton/bigbluebutton#11050]
**Is your feature request related to a problem? Please describe.**
I believe BBB is the best solution for online lecture over other services such as Zoom or Teams. However, it's still hard to convince ordinary people who do not have a good IT knowledge and skills. The largest problem is that BBB cannot handle the animated effects and embedded videos of a Powerpoint file. I can ask the people to remove the effect, upload the videos to Youtube beforehand, etc, but these tedious procedures very efficiently discourage them to use BBB. People do not always prioritize the quality of the presentation (in which BBB has an advantage), they like an "easy-to-prepare" solution. You may say they can still use shared screen function, but I don't think this feature of BBB overperforms Zoom or Teams, especially the sound not being shared in BBB yet. 

**Describe the solution you'd like**
BBB reads .pptx file and
1. extract media files and copy them to an appropriate directory to share with participants, probably by using the framework of #8753 , and write the local address into the place of PDF where the video has been embedded so that it automatically appears as a quick video link feature proposed in #9609 or #10588 .
2. split the animations automatically into multiple different slide using a Libreoffice macro or plugin such as https://gist.github.com/maffoo/977752 or https://github.com/monperrus/ExpandAnimations .

**Additional context**
I know there will be a couple of new problems arising by this change. Users will anyway complain about the quality of conversion. Better to make this feature optional at the beginning. 
I am considering to work on it but before starting, I would like to know if there is already a work on it and the project feasibility evaluated by expert developers (which I am not).

Related to #9734 and #10403 (and probably more).

**Comment by basisbit** [comment]
this is literally a duplicate of 9734 and 10403 and 2489, which you already linked to...
